Homework3: Andrei Bartosh

sharp-0000ff opened the pull request.

sharp-0000ff committed:
release 0.3

sharp-0000ff committed:
release 0.3

belovmd reviewed (changes requested):
Thank you for your work. 
Please address next paragraphs from review notes [Review Notes](https://github.com/belovmd/it-academy-python-spring/blob/master/homework_review/Homework3.md):
- 15
- 16
- 4
- 11

sharp-0000ff committed:
Make fixes after Code Review
Used i % 15 instead of i % 3 and i % 5
Used if not value instead of value == 0
Task 4 and Task 5: changed complexity algorithm from O(n**2) to O(n)

belovmd reviewed (commented):
9,5
